app/views/reports/standard_form_report/subset/_table_headers.html.erb

```diff
@@ -0,0 +1,33 @@
+<% # do cluster header row %>
+<tr class="cluster_header">
+  <td></td>
+
+  <% # if full width display type with responses, header cell must be extra wide %>
+  <% if cluster.display_type == :full_width %>
+    <td <%= @report.references? ? "colspan=#{result_cols + 1}" : "colspan=#{result_cols + 2}" %> >
```

Comment:
Simplify to `colspan="<%= result_cols + (@report.references? ? 1 : 2) %>"`. 

Note the simplification theme coming up again here. Think back to algebra class -- it's like factoring an equation :D

Reply:
kk, done